File: apprefurbished/src/main/java/com/androidcollider/easyfin/managers/ui/shake_edit_text/ShakeEditTextModule.java
package com.androidcollider.easyfin.managers.ui.shake_edit_text;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ShakeEditTextModule {

    @Provides
    ShakeEditTextManager provideShakeEditTextManager() {
        return new ShakeEditTextManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/ui/shake_edit_text/ShakeEditTextModule.kt
package com.androidcollider.easyfin.common.managers.ui.shake_edit_text

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class ShakeEditTextModule {
    @Provides
    fun provideShakeEditTextManager(): ShakeEditTextManager {
        return ShakeEditTextManager()
    }
}